CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Change Log
 
+## [0.8.0](https://github.com/redhat-developer/vscode-xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * Find definition for start/end tag element. See[#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Improve Source information from completion. See[#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * Completion should support markdown for documentation. See[#526](https://github.com/angelozerr/lsp4xml/issues/526)
```

Comment:
Markdown support for completion documentation

Reply:
I remobed this issue since https://github.com/redhat-developer/vscode-xml/issues/32 covers it